packages/codemods/src/transform.ts

```diff
@@ -0,0 +1,42 @@
+import path from 'path';
+import yargs from 'yargs';
+import { execSync } from 'child_process';
+
+const transformerDirectory = path.join(__dirname, 'transforms');
+const jscodeshiftExecutable = require.resolve('.bin/jscodeshift');
+
+interface argsType {
+  path: string;
+  transform: string;
+}
+
+export const yargsParse = (args: string[]): argsType => {
+  return yargs([])
+    .help()
+    .exitProcess(false)
+    .option('path', {
+      alias: 'p',
+      type: 'string',
+      description: 'Path that transform should be run over',
+      normalize: true,
+    })
+    .option('transform', {
+      alias: 't',
+      type: 'string',
+      description: 'Name of transform to run',
+      choices: ['button-v0-to-v1'],
+    })
+    .demandOption(['path', 'transform'])
+    .parse(args);
+};
+
+export const transform = (args: argsType) => {
+  const codeshiftArgs = [];
+
+  codeshiftArgs.push('-t', path.join(transformerDirectory, args.transform + '.js'));
```

Comment:
Should this be .ts? I see button-v0-to-v1.ts, not .js

Reply:
When the code is running, it's using the transpiled js, so from its perspective everything is a js file

Reply:
We publish the transpiled .js files, not the .ts files directly

Reply:
Ah I see okay, thanks for explaining